mpas_xarray.py

```diff
@@ -54,6 +54,34 @@ def preprocess_mpas(ds, yearoffset=1700): #{{{
 
     return ds #}}}
 
+def preprocess_mpas_timeSeriesStats(ds, yearoffset=1849): #{{{
+    """
+    Builds corret time specification for MPAS time series stats am fields,
+    allowing a year offset because the time must be betwee 1678 and 2262 
+    based on the xarray library.
+
+    Milena Veneziani
+    04/18/2016
+    """
+
+    daysSinceStart = ds.timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1
+    time = [datetime.datetime(yearoffset,12,31) + datetime.timedelta(x) for x in daysSinceStart.values]
```

Comment:
May also want to include month and day offsets as arguments to `preprocess_mpas_timeSeriesStats` above.  I think this would be helpful for generality and to avoid "magic" numbers.